svo (actions dictionary).xml

```diff
@@ -1772,6 +1772,32 @@ if not next(svo.dict) then
         oncompleted = function() svo.rmaff('justice') end,
       }
     },
+    pyre = {
+      herb = {
+        isadvisable = function()
+          return (affs.pyre and (not (svo.haveskillset('discipline') or svo.haveskillset('malignity') or svo.haveskillset('valour') or svo.haveskillset('domi
```

Comment:
`'domi` is split across two lines by accident, if you bring it back on one line, the syntax error will go away.

> Error in Script 'Dictionary of actions (affs/defs/misc)': [string "-- Svof (c) 2011-2018 by Vadim Peretokin..."]:1767: unfinished string near ''domi'

Reply:
I was just looking at that and wondering why. You're the man.